src/components/Organization/projects.json

```diff
@@ -255,6 +255,6 @@
 		"repo": "webpack-contrib/css-minimizer-webpack-plugin",
 		"npm": "css-minimizer-webpack-plugin",
 		"description": "Optimize and minify your CSS using cssnano.",
-		"maintainer": ""
+		"maintainer": "cap-Bernardito"
```

Comment:
I think better to use `webpack-contrib` here, because it supported by multiple developers

Reply:
Yeah, also this should support an array of contributors

Reply:
ok given its not supported lets merge, it was sufficient previously